### PR TITLE
Add enterprise portal + update smoke tests

### PR DESCRIPTION
**Pull request by zalbiraw**
## Description
Adds ability to launch an enterprise portal to tyk-pro

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [x] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by zalbiraw** [comment]
What the PR is missing: 
- portal connection string should be a secret
- bootstrap script should bootstrap portal
- sqlite3 is not working. Seems to be a permission issue in creating the db file.

**Comment by zalbiraw** [comment]
@caroltyk here is the [PR](https://github.com/TykTechnologies/tyk-docs/pull/1968) for the docs.

**Comment by zalbiraw** [comment]
sqlite3 not working is expected. You need to mount a file on the file system for that to work. If we allow sqlite3 to create its own file on the container it will cause a lot of issues with data loss and data not being copied to different containers. So if someone wants to use sqlite3 they will need to create their own volumes and attach them to the containers. 

PR is now ready for test.

**Comment by zalbiraw** [comment]
Tested on minikube, EKS,  and OpenShift with mysql and postgres.

**Comment by zalbiraw** [comment]
Updated PR and smoke tests to the latest version.

**Comment by zalbiraw** [comment]
Please don't merge before 
https://github.com/TykTechnologies/tyk-helm-chart/pull/220
https://github.com/TykTechnologies/tyk-helm-chart/pull/226
https://github.com/TykTechnologies/tyk-helm-chart/pull/227

**Comment by zalbiraw** [comment]
Need to find a fix for the enterprisePortal testing with PVC and wait on @andrei-tyk's bootstrapper to merge https://github.com/TykTechnologies/tyk-k8s-bootstrap/pull/1

**Comment by zalbiraw** [comment]
@andrei-tyk this should work ootb as long as you push your PR to the latest docker tag!

File: tyk-pro/values.yaml [hunk 1]
```diff
@@ -212,6 +212,10 @@ dash:
   # and operatorSecret are set to true.
   # Set false to disable creating a secret for Tyk Operator.
   operatorSecret: true
+  # It creates a secret for Tyk Enterprise Developer Portal along with bootstraping dasboard. Enterprise Developer Portal
```
Comment (komalsukhani) [inline tyk-pro/values.yaml:215]:
```suggestion
  # It creates a secret for Tyk Enterprise Developer Portal along with bootstraping dashboard. Enterprise Developer Portal
```

**Review by andrei-tyk — APPROVED** [review]

**Pull request merged**